x-pack/plugins/apm/public/context/url_params_context/helpers.ts

```diff
@@ -23,18 +23,7 @@ export function getExactDate(rawDate?: string, options = {}) {
   if (rawDate) {
     const isRelativeDate = rawDate.substring(0, 3) === 'now';
     if (isRelativeDate) {
-      const isSubtractingDate = rawDate.indexOf('-') > 0;
-      const isRoundingDate = rawDate.indexOf('/') > 0;
-
-      const rawDateWithouRounding =
-        // When relative time is subtracting a period and rounding the result (e.g. now-24h/h)
-        // removed the rounding part in order to get the exact time.
-        // This is needed because of of "Today"(now/d) and "This week"(now/w) options, it rounds the values up and down
-        // so the exact time is the rounded value.
-        isSubtractingDate && isRoundingDate
-          ? rawDate.substring(0, rawDate.indexOf('/'))
-          : rawDate;
-
+      const rawDateWithouRounding = rawDate.replace(/\/(\w)$/, '');
```

Comment:
Please keep the code comment. Also, perhaps we can make the regex more strict and only replace /s /m /h /w

```suggestion
      // remove rounding from relative dates "Today" (now/d) and "This week" (now/w)
      const rawDateWithouRounding = rawDate.replace(/\/([smhdw])$/, '');
```

Reply:
Hmm.. Looking closer at this I dont understand why we need this in the first place. 

```js
const datemath = require('@elastic/datemath')

// This will return the start of the current day
const start = datemath.parse('now/d'); // Moment<2021-06-14T00:00:00+02:00>

// this will return the end of the current day
const end = datemath.parse('now/d', {roundUp: true}) // Moment<2021-06-14T23:59:59+02:00>
```

In the above case `end - start` should always be exactly 86399999 ms ~ 24 hours (minus 1 millisecond.

Reply:
I thought the problem with this rounding was, that in certain cases the rounding would cause the duration between start and end to be larger than 24 hours.

Reply:
I just tried this out locally, and there is no need to do any of this, if you just remove the `isRangeToNow` check.